internal/local/app_test.go

```diff
@@ -68,19 +69,57 @@ func TestNewApp(t *testing.T) {
 func TestLoadApp(t *testing.T) {
 	wd, wdErr := os.Getwd()
 	assert.Nil(t, wdErr)
-
 	testRoot := wd
-	projectRoot := filepath.Join(testRoot, "testdata", "full_project")
 
-	expectedAppLocal := App{
-		RootDir: projectRoot,
-		Config:  FileConfig,
-		AppData: fullProject,
+	t.Run("loading an app with a valid path should succeed", func(t *testing.T) {
+		projectRoot := filepath.Join(testRoot, "testdata", "full_project")
+
+		expectedAppLocal := App{
+			RootDir: projectRoot,
+			Config:  FileConfig,
+			AppData: fullProject,
+		}
+
+		app, appErr := LoadApp(projectRoot)
+		assert.Nil(t, appErr)
+		assert.Equal(t, expectedAppLocal, app)
+	})
+
+	t.Run("loading an app from an invalid path should error", func(t *testing.T) {
+		invalidAppPath := filepath.Join(testRoot, "testdata")
+
+		app, appErr := LoadApp(invalidAppPath)
+		assert.Equal(t, appErr, ErrFailedToFindApp(invalidAppPath))
+		assert.Equal(t, App{}, app)
+	})
+}
+
+func TestLoadConfig(t *testing.T) {
+	wd, wdErr := os.Getwd()
+	assert.Nil(t, wdErr)
+
+	for _, config := range []struct {
+		version realm.AppConfigVersion
+		file    File
+	}{
+		{realm.AppConfigVersion20180301, FileStitch},
+		{realm.AppConfigVersion20200603, FileConfig},
+		{realm.AppConfigVersion20210101, FileRealmConfig},
+	} {
+		t.Run(fmt.Sprintf("should successfully load config file with version %d", config.version), func(t *testing.T) {
+			projectRoot := filepath.Join(wd, "testdata", config.version.String(), "local")
+			testApp := App{RootDir: projectRoot, Config: config.file}
+
+			assert.Nil(t, testApp.LoadConfig())
```

Comment:
are there tests/asserts on `testApp` we could include here just for completeness?